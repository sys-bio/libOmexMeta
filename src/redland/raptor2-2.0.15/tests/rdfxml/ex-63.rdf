<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF xmlns:dcterms="http://purl.org/dc/terms/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
   <rdf:Description rdf:about="http://example.org/doc/pmc/2751467">
      <dcterms:description rdf:parseType="Literal">
         <div xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
            <p b="foo" a="bar">123</p>

         </div>
      </dcterms:description>
   </rdf:Description>
</rdf:RDF>
