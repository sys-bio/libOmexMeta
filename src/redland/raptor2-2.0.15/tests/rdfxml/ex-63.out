<http://example.org/doc/pmc/2751467> <http://purl.org/dc/terms/description> "\n         <div xmlns=\"http://www.w3.org/1999/xhtml\" xml:lang=\"en\">\n            <p a=\"bar\" b=\"foo\">123</p>\n\n         </div>\n      "^^<http://www.w3.org/1999/02/22-rdf-syntax-ns#XMLLiteral> .
